modules/ipc/include/hephaestus/ipc/zenoh/action_server/pollable_action_server.h

```diff
@@ -133,7 +133,8 @@ PollableActionServer<RequestT, StatusT, ReplyT>::PollableActionServer(SessionPtr
             }
 
             if (status_.has_value()) {
-              status_publisher.publish(*std::move(status_));
+              status_publisher.publish(std::move(*status_));
```

Comment:
I would remove std::move alltogether as the publish function anyway takes a reference